src/Core/hco_datacont_mod.F90

```diff
@@ -190,6 +190,7 @@ SUBROUTINE DataCont_Init( Dct, cID )
     Dct%cName        = ''
     Dct%spcName      = ''
     Dct%ScalID       = -999
+    Dct%srcGMaskID   = -999
```

Comment:
@1Dandan: Use 
```F90
Dct%srcGMaskId = -999.0_sp
``` 
in order to avoid repeated casting from integer to real on each (I,J,L) iteration.